packages/navigator/src/browser/navigator-model.ts

```diff
@@ -132,6 +134,13 @@ export class FileNavigatorModel extends FileTreeModel {
         return undefined;
     }
 
+    selectNode(node: SelectableTreeNode): void {
+        if (FileNode.is(node) && this.preferences['navigator.enablePreview']) {
+            open(this.openerService, node.uri, {preview: true});
```

Comment:
you should be able to pass `mode: ' reveal'` here to preserve focus

Reply:
Ah! Thanks for this! To get consistent focus, I used {mode: reveal} in the widget manager and made a few other minor changes.

Reply:
We need reveal here also, otherwise if editor preview is disabled, then focus is again stolen from the navigator.